src/cmakeBuildRunner.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Module for running CMake Build Task
+ */
+
+import * as vscode from 'vscode';
+import * as nls from 'vscode-nls';
+import * as proc from '@cmt/proc';
+import * as util from '@cmt/util';
+
+nls.config({ messageFormat: nls.MessageFormat.bundle, bundleFormat: nls.BundleFormat.standalone })();
+
+export class CMakeBuildRunner {
+
+    private taskExecutor: vscode.TaskExecution | null = null;
```

Comment:
Please use `undefined` instead of `null`  (applies to the whole file).  We will be trying to switch all `null` usages to `undefined` in the future.

Reply:
Can do.
@elahehrashedi  please let me know if I should do it or you prefer to take over.

Reply:
I think @elahehrashedi was planning to address some of your comments. She can fix this one too.